styles.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+ul,
+a {
+  text-decoration: none;
+  list-style: none;
+}
+
+.container {
+  margin: auto;
+  width: 375px;
+  height: 718px;
+  border: 1px solid rebeccapurple;
+  border-bottom-left-radius: 7%;
+}
+
+aside ul {
+  display: flex;
+  padding-top: 10px;
+}
+
+.header ul {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px 10px 12px 40px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 8px 30px;
+}
+
+.logo {
+  width: 100px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+.hero {
+  background-image: url(./assets/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  height: auto;
+}
+
+.author {
+  padding: 80px 20px 0 12px;
+  width: 330px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 56px;
+  line-height: 55px;
+  color: #172b4d;
+}
+
+.bio {
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 26px;
+  padding: 20px 10px 8px 10px;
+  color: #344563;
+}
+
+.connect_button {
+  text-transform: uppercase;
+  color: #7f8cff;
+  font-size: 20px;
+  padding: 0 10px 0 10px;
+}
```

Comment:
- Please, let's improve our design by using the correct font family, font-weight, font-size, color, etc here to meet the design as much as we can below:

*design template*

![Screenshot from 2022-10-28 01-19-15](https://user-images.githubusercontent.com/59397159/198421562-e71a1149-16f2-423c-9f29-1ec3d9091d1f.png)

*your design*

![Screenshot from 2022-10-28 01-27-17](https://user-images.githubusercontent.com/59397159/198421755-9a24551d-e78f-4412-b83b-71937c8f8c9f.png)